solutions/usaco-570.mdx

```diff
@@ -2,72 +2,353 @@
 id: usaco-570
 source: USACO Silver 2015 December
 title: Switching on the Lights
-author: Óscar Garries
+author: Akshar Yeccherla
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_lightson_silver_dec15.html)
 
+## Implementation
```

Comment:
you should also probably add a short editorial above the implementation tags.

format:
```
## Explanation

[editorial content]
```